docs/user-guide/event-data.ipynb

```diff
@@ -125,11 +130,26 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "a = sc.DataArray(data=var, coords={\n",
-    "    'x': sc.Variable(['x'], values=np.arange(4.0)),\n",
-    "    'y': var})\n",
-    "var['x', 0].values = np.arange(7)\n",
-    "var['x', 3].values = np.ones(2)\n",
+    "weights = sc.Variable(\n",
+    "    dims=['x'],\n",
+    "    shape=[4],\n",
+    "    unit=sc.units.counts,\n",
+    "    dtype=sc.dtype.event_list_float64,\n",
+    "    variances=True)\n",
+    "weights['x', 0].values = np.ones(6)\n",
+    "weights['x', 1].values = np.ones(1)\n",
+    "weights['x', 2].values = np.ones(0)\n",
+    "weights['x', 3].values = np.ones(6)\n",
+    "weights['x', 0].variances = np.ones(6)\n",
+    "weights['x', 1].variances = np.ones(1)\n",
+    "weights['x', 2].variances = np.ones(0)\n",
+    "weights['x', 3].variances = np.ones(6)\n",
+    "\n",
+    "a = sc.DataArray(\n",
+    "    data=weights,\n",
+    "    coords={'x': sc.Variable(['x'], values=np.arange(4.0)),\n",
+    "            'time': var})\n",
+    "a.coords['time'].unit = sc.units.us # micro second\n",
```

Comment:
Not new here, but might be worth explaining what users are seeing from `show`, i.e. 2 yellow arrays for sparse values and variances.

Reply:
Done.